Vagrantfile

```diff
@@ -21,7 +23,7 @@ Vagrant.configure(2) do |config|
   config.vm.box = "centos7"
   config.vm.box_url = "http://cloud.centos.org/centos/7/vagrant/x86_64/images/CentOS-7-x86_64-Vagrant-1608_01.LibVirt.box"
 
-  if Vagrant.has_plugin?("vagrant-cachier") and $cache_rpm then
+  if Vagrant.has_plugin?("vagrant-cachier") and $cache_rpm and $use_nfs then
```

Comment:
adding $use_nfs here means that it breaks rpm caching for my dev setup.  Why is this a requirement for rpm caching now?

Reply:
Well, when I tested it, I saw that vagrant-cachier was trying to use NFS even though I was using `rsync` for source sync.
I need to prevent that because its kinda naive about how its sets up NFS (It writes to /etc/eports and restarts the NFS service via `systemctl`, both of which cannot work in a chroot or a container without serious trickery) and it just makes Vagrant get stuck whiting forever for volumes to show up when it fails to do that.